source/core/src/main/com/deco2800/game/components/story/StoryConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.deco2800.game.components.story;
+
```

Comment:
Similarly for `StoryBase.java` - probably good to have an explanation like you've already written - 'Stores raw data and methods needed to create the entities. Each story base must have a config.'